lib/analytics/course_students_csv_builder.rb

```diff
@@ -17,6 +22,28 @@ def generate_csv
 
   private
 
+  def populate_created_articles
+    # A user has created an article during the course if
+    # the user is in the user_ids of new_articles_courses
+    # has a revision with new_article: true for that article
+    @course.new_articles_courses.pluck(:article_id, :user_ids).each do |article_id, user_ids|
+      user_ids.each do |user_id|
+        @created_articles[user_id] += 1 if article_creator?(article_id, user_id)
+      end
+    end
+  end
+
+  def article_creator?(article_id, user_id)
+    @revisions[article_id] == user_id
+  end
+
+  def populate_edited_articles
+    # A user has edited an article if the user is in the user_ids list of edited_articles_courses
+    @course.edited_articles_courses.pluck(:article_id, :user_ids).each do |_article_id, user_ids|
```

Comment:
Is it necessary to pluck the article_id here?

Reply:
I think so because pluck is using select DISTINCT so if I leave out article_id it'll return just the ammount of combinations of user id's lists

![image](https://user-images.githubusercontent.com/24531739/103444573-4bab7a80-4c61-11eb-8aca-26dc5beee1ba.png)

Reply:
cool